Kernel/Graphics/Intel/Definitions.h

```diff
@@ -10,30 +10,14 @@
 
 namespace Kernel::IntelGraphics {
 
-enum class RegisterIndex {
+enum class GlobalGenerationRegister {
     PipeAConf = 0x70008,
     PipeBConf = 0x71008,
-    GMBusData = 0x510C,
-    GMBusStatus = 0x5108,
-    GMBusCommand = 0x5104,
-    GMBusClock = 0x5100,
```

Comment:
I don't understand why in  `Kernel/Graphics: Prepare Intel driver to support more generations` we don't have any enums for GMBus registers. If GMBus IP being at `0x5100` is specific to Gen4, can't we create a `Gen4RegisterIndex` enumeration to hold it?

Reply:
Because it's not really specific to Gen4. GMBus chips could be in Gen4, Gen6, Gen9 and Gen9.5 of Intel graphics.